Jenkinsfile

```diff
@@ -5,23 +5,17 @@ properties([
         buildDiscarder(logRotator(numToKeepStr: '10', artifactNumToKeepStr: '2')),
 ])
 
-node('linux') {
+node('maven-8') {
```

Comment:
For now, trusted.ci only have a `maven-11` label: https://github.com/jenkins-infra/jenkins-infra/blob/production/hieradata/clients/trusted-ci.yaml#L49 (because JDK 11 is the default used so the default JAVA_HOME points toward JDK8).

I propose that we keep the current pipeline and close this PR as the instabilities of update_center2 jobs, on both ci.jenkins.io and trusted.jenkins.io were due to JFrog's incident.

@MarkEWaite can you confirm that my message makes sense and that I did not forget anything?

Reply:
Irrelevant because the job on trusted.ci does not use this Jenkinsfile.

Reply:
oh, what is the pipeline used on trusted ? (but good to know \o/)

Reply:
Freestyle job.

Reply:
<insert screaming emoji here>

Reply:
Closing the pull request because the issues were unrelated to this change.  Thanks @dduportal